src/React.Core/IReactSiteConfiguration.cs

```diff
@@ -114,16 +114,14 @@ public interface IReactSiteConfiguration
 		IReactSiteConfiguration SetMaxUsagesPerEngine(int? maxUsagesPerEngine);
 
 		/// <summary>
-		/// Gets or sets whether the MSIE engine should be used if V8 is unavailable.
+		/// Gets or sets the name of the preferred JavaScript engine.
 		/// </summary>
-		[Obsolete("This should be managed in the JavaScriptEngineSwitcher configuration instead")]
-		bool AllowMsieEngine { get; set; }
+		string PreferredEngineName { get; set; }
```

Comment:
Can we just use the JavaScriptEnginerSwitcher property for this? I think it has a preferred/default engine setting too.